react/react-training/src/app/screens/Error404/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { ERROR_404 } from '@constants';
+
+import cry from './assets/404.png';
+import styles from './styles.scss';
+
+function Error404() {
+  return (
+    <div>
+      <h2 className={styles.title}>{ERROR_404}</h2>
+      <p className={styles.image}>
+        <img alt={ERROR_404} height={cry.height} src={cry} width={cry.width} />
```

Comment:
Isn't a good practice to assign the properties `width` and `height` here, please move it to `styles` file